framework/src/arastorage/index_manager.c

```diff
@@ -275,6 +275,7 @@ db_result_t index_load(relation_t *rel, attribute_t *attr)
 		api = find_index_api(index->type);
 		if (api == NULL) {
 			DB_LOG_E("DB: No API for index type %d\n", index->type);
+			memb_free(&index_memb, index); 
```

Comment:
Basically we brought arastorage from contiki.
contiki using memblock because it doesn't provides malloc.
Could you please check this memb can be changed to malloc in next step?

Reply:
OK, I will check this requirement and report it later.